packages/prime/src/prime_cli/commands/env.py

```diff
@@ -1927,6 +1926,106 @@ def delete(
         raise typer.Exit(1)
 
 
+def _check_environment_installed(env_name: str, required_version: str | None = None) -> bool:
```

Comment:
I think we can actually skip this step since `prime env install` is idempotent, and this `check_code` seems finnicky.

This is also how `prime-rl` handles the auto-install btw, see here: https://github.com/PrimeIntellect-ai/prime-rl/blob/04a32f747a6365e1214e4977776cfcbd0cc6fec5/src/prime_rl/eval/eval.py#L35-L38

Reply:
okay then in that case, we should use something like `uv pip show` or a `uv pip list --format=json` where we parse the output.

because in the `check_code`, we're checking if the `load_environment` works _in addition_ to whether it's been installed or not.

so if the `load_environment` fails due to syntax issues, broken imports, etc. then we assume that the package is just not installed, which is incorrect.

Reply:
Good point, resolved now.